integration/docker/Dockerfile

```diff
@@ -32,7 +32,10 @@ RUN if [ ${ENABLE_DYNAMIC_USER} = "true" ] ; then \
        chmod -R 777 /opt/* ; \
     fi
 
-FROM openjdk:8-jdk-alpine
+# instead of ubuntu:16.04, use alluxio/alluxio-ubuntu:ubuntu1604-customize
+# applies optimizations in libfuse on top of ubuntu:16.04
+# it is based off cheyang/fuse2:ubuntu1604-customize
+FROM alluxio/alluxio-ubuntu:1604-customize
```

Comment:
I think there are some differences between debian and alpine. For example, it uses 'apt' in debian while 'apk' in alpine

Reply:
commented out